support-frontend/package.json

```diff
@@ -76,6 +76,7 @@
     "@guardian/src-choice-card": "^1.6.0",
     "@guardian/src-foundations": "^1.6.0",
     "@guardian/src-icons": "^1.6.0",
+    "@guardian/src-link": "^2.0.0",
```

Comment:
This is probably fine as it is, but in general I'd recommend keeping the version numbers of all `src-*` components in sync because I [can't guarantee compatibility](https://www.theguardian.design/2a1e5182b/p/876251-getting-started/t/9634cf) across major versions 🙏